plottable.css

```diff
@@ -124,6 +124,7 @@ svg.plottable {
 }
 
 .plottable .legend text {
+  fill: #32313F;
```

Comment:
Sets the `Legend` text to be the same color as used on `Axis` and `Label`. I can get Design to officially sign off on it, but the goal is consistency.